doc/documentation.tex

```diff
@@ -466,6 +417,105 @@ \subsubsection{Methods comparison}
 
 \subsection{Noise and outliers tolerance}
+We tested our methods using the clouds and conditions that may be closer to reality. That is, we used clouds that are missing points or even entire part and we corrupted clouds by noise and outliers. For this purpose we have used CPU versions of algorithms. For ICP and NICP \textit{policy} was set to \textit{parallel} and \textit{approximation-type} to \textit{none}. For CPD \textit{approximation-type} was set to \textit{hybrid}, mainly to achieve faster computation times. Many tests were conducted and it turned out that results are highly dependent on configuration. Therefore, we will show the most interesting results and tests that have shown the most consistent characteristics. All results are available in spreadsheet kept on Github: \href{https://github.com/Sliwson/cuda-slam/tree/cpd-documentation/doc/noise}{https://github.com/Sliwson/cuda-slam/tree/cpd-documentation/doc/noise}. Also in directory \textit{config} there are all test configurations.\\
+After testing we came to several conclusions. Firstly, the ICP algorithm coped the worst with almost all test sets. The main reason can be cloud sizes used in tests, which occurred to be often too big for ICP capabilities. Algorithm had also trouble with noise. Even low level noise such as \textit{noise-affected-points-before}$=0.1$ and \textit{noise-intensity-before}$=0.05$ may cause convergence failure. Secondly, increasing $max-iterasions$ can help achieving more exact results. CPD algorithm mostly achieved correct results. Main key to CPD convergence is to select parameters value correctly. The most important one is \textit{cpd-weight} which describes how many points 
+are noise and outliers. Difference of $0.3$ can lead to significantly different result. Second CPD parameter is \textit{cpd-const-scale}. Setting it to \textit{true} may help achieving better results when we know that the scale did not change after transformation. Especially it helps when clouds contain a lot of noise and outliers. Also, decresing \textit{cpd-tolerance} and \textit{converge-epsilon} (the latter applies also to ICP and NICP) may result in increasing iterations count and lead to more exact result.\\
+All algorithms had tendency to match areas where the point density was the highest. An example of this behavior may be observed using \textit{Config21.json}. It uses two "airbus" clouds with decreased sizes and added noise and outliers. Result achieved by algorithms are presented below and vary among each other.
```

Comment:
- point density -> points density
- do we name config files using upper case? maybe we should go for lower case if it is not a lot of work

Reply:
Done